presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -240,6 +240,7 @@ SystemConfig::SystemConfig() {
           BOOL_PROP(kExchangeEnableConnectionPool, true),
           BOOL_PROP(kExchangeEnableBufferCopy, true),
           BOOL_PROP(kExchangeImmediateBufferTransfer, true),
+          NUM_PROP(kExchangeMaxBufferSize, 32UL << 20),
```

Comment:
This line requries MB in unit. Otherwise quereis fail with 

presto:di> select * from tpch.sf1.orders limit 2
Query 20251101_231501_00108_hcf8c failed: Non-whitespace character found after end of conversion: "MB"